test/src/unit/repo_test_utils.h

```diff
@@ -197,3 +199,133 @@ static std::string getRandomString(const uint32_t &iLen)
 
 	return sStr;
 }
+
+namespace repo
+{
+namespace test
+{
+	class TestLogging
+	{
+	public:
+
+		static void printTestTitleString(std::string title, std::string description)
+		{
+			repoInfo << "============" << title << "============";
+			repoInfo << description;
+		}
+
+		static void printSubTestTitleString(std::string title)
+		{
+			repoInfo << "------------" << title << "------------";
+		}
+
+		static std::string getStringFromRepoErrorCode(int repoErrorCode)
```

Comment:
(just saw this whilst replying to the other question)
@har00n-haider Does this need to exist?

Again, this goes back to the preference of keeping logging precise and minimal. If an error was triggered gtest will tell us what was the expected error code is suppose to be. There's not a lot of need to print this.

This also has a danger of going out of sync with error_codes.h

Reply:
As with above I'll axe this.